src/Cli/dotnet/commands/dotnet-tool/update/ToolUpdateGlobalOrToolPathCommand.cs

```diff
@@ -22,12 +22,13 @@ internal class ToolUpdateGlobalOrToolPathCommand : CommandBase
     {
         private readonly CreateShellShimRepository _createShellShimRepository;
         private readonly CreateToolPackageStoresAndDownloaderAndUninstaller _createToolPackageStoreDownloaderUninstaller;
-        private readonly ToolInstallGlobalOrToolPathCommand _toolInstallGlobalOrToolPathCommand;
+        private readonly Lazy<ToolInstallGlobalOrToolPathCommand> _toolInstallGlobalOrToolPathCommand;
```

Comment:
What's the reason for changing this to be lazily initialized?

I see that ToolUpdateLocalCommand uses a similar pattern, but I'm not sure the reason for that.  It doesn't seem to be necessary, but maybe I'm missing something, or maybe there used to be a reason but the code changed since then.

Reply:
Thanks for noticing this! I checked the git blame for the Local counterpart. The commit was in 2019 and there was no signficiant reasons for writing it into `Lazy` as the code was initially written into `Lazy` rather than changed to `Lazy`.

Therefore, I removed the `Lazy` initialization. Thanks again!